### ISSUE TITLE
Allow moving scrubber by keyboard when not in focus

### ISSUE THREAD

**Issue by Arnei** [elan-ev/opencast-editor#265]
Currently the scrubber can only be moved by keyboard shortcuts when it is focused. Instead, it should be possible to also move the scrubber when it is not focused. -Wien

**Comment by lkiesow** [comment]
This is more or less a duplicate of #300 and should be solved by using arrow keys only
